Makefile

```diff
@@ -410,7 +416,9 @@ ifneq ($(DESTDIR),)
 endif
 	@$(MAKE) -C $(BUILDROOT) -f $(JULIAHOME)/Makefile install
 	cp $(JULIAHOME)/LICENSE.md $(BUILDROOT)/julia-$(JULIA_COMMIT)
-ifneq ($(OS), WINNT)
+	# Run fixup-libgfortran on all platforms but Windows and FreeBSD
+	# On FreeBSD we're getting and fixing these libraries earlier
+ifeq (,$(findstring $(OS),FreeBSD WINNT))
```

Comment:
I would explain _why_ we fix the libraries earlier.  Something like "`FreeBSD` doesn't link to the compiler-provided `libgcc_s` by default, which causes problems, so we copy our compiler's `libgcc_s` into our `lib` directory at the beginning of the build" or something like that.